docs/user/kubevirt/install_ipi.md

```diff
@@ -0,0 +1,156 @@
+   - [KubeVirt](https://kubevirt.io/) installed in the cluster
+   - [Multus-cni](https://01.org/kubernetes/building-blocks/multus-cni) plugin installed in the cluster<br/>
+
+   prerequisite for OpenShift:
+   - [OpenShift Virtualization](https://www.openshift.com/learn/topics/virtualization/)
+
+2. Create the network that would be used by the tenant cluster:
+   - Create a bridge on the infra cluster nodes
+   - Create multus network (Network Attachment Definition) on the bridge
+
+3. Allocate 2 IPs within the cluster network:
+    - IP for the internal kubernetes api, that all components will interact with 
+    - IP for the Ingress, the load balancer in front of the cluster apps 
+
+    To work with this provider one must supply 2 IPs that are not related to any MAC in the environment, where the cluster will run. Those IPs will be active by keepalived, on, initially the bootstrap machine, and then the masters, after a fail-over, when the bootstrap is killed. 
+
+4. Name resolution of `api_vip` from your installing machine 
+The installer must resolve the `api_vip` during the installation, as it will interact with the API to follow the cluster version progress. 
+
+
+## Minimum resources
+
+The default master/worker:
+- 4 CPUs
+- 16 RAM
+- 120 GB disk
+
+For 3 masters/3 workers, the target Cluster **must have at least**:
+- 96RAM
+- 24vCPUs
+- 720GiB storage
+
+> Worker count can be reduced to 2 in `install-config.yaml` in case needed.
+
+The cluster will create by default 1 bootstrap, 3 master, and 3 workers machines. 
+By the time the first worker is up the bootstrap VM should be destroyed, and this is included in the minimum resources calculation.
+
+
+## Install 
+
+### Minimum permission for installation
+
+In order to be able to complete full installation process, the user running the installer must have privilaged for minimum permission.<br/>
+Please refer the the [Role yaml](./role.yml) to view the list of the required permissions.<br/>
+This role can be added to the cluster, and then binded to the user which running the installer, it can be done using the following ways:
+- Create RoleBinding which bind the role to the user in the specific namespace
+- If using Openshift as InfraCluster, run:<br/>
+`oc adm policy add-role-to-user <role> <user> -n <namespace>`
+
+
+### kubevirt-credentials
+
+Before you start the tenant cluster installation, make sure to login the infra cluster, one of the ways:
+
+1. For configuration, kubectl looks for a file named config in the $HOME/.kube directory. You can specify other kubeconfig files by setting the KUBECONFIG environment variable or by setting the --kubeconfig flag.
+2. Using the OpenShift oc command-line client (`oc login` command)
+https://docs.openshift.com/container-platform/4.6/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands
+
+### Bootstrap VM
+
+The bootstrap will perform ignition fully and will advertise the IP in the pre-login msg.
```

Comment:
I know this sentence is copied from oVirt manual (-:. but it makes no sense to me ...

Reply:
Removed this "Bootstrap" section from the documentation